### ISSUE TITLE
[Feature request] Option to render .qmd files to specified format after export

### ISSUE THREAD

**Issue by machado-t** [AndreasThinks/obsidian-to-quarto-exporter#4]
In my workflow, I use the plugin to convert Obsidian markdown files to Quarto .qmd files, and then I open RStudio to render the .qmd file to my target format, typically .pptx.

It would be helpful to have an option/command in the plugin to automate this process, allowing the .qmd file to be rendered to the specified output format immediately after export to .qmd. This feature could save time and reduce the number of steps in my workflow.

Thanks!

**Comment by AndreasThinks** [comment]
So to be sure I've understood here, you're using Obsidian to covert to QMD, then Quarto to convert from QMD to pptx, and you'd like to have a "one shot"? 

Not quite sure how this could be achieved (other than maybe obsidian forcing a "quarto render" command or similar)?

**Comment by machado-t** [comment]
> So to be sure I've understood here, you're using Obsidian to covert to QMD, then Quarto to convert from QMD to pptx, and you'd like to have a "one shot"?

Exactly. Currently, I'm using your plugin specifically for converting md to pptx. Other plugins (e.g. pandoc plugin) can do this direct conversion but lack the features and flexibility of Quarto (e.g. speaker notes, templates, other options). I may use it in the future for other formats other than pptx. 

> Not quite sure how this could be achieved (other than maybe obsidian forcing a "quarto render" command or similar)?

As for the technical side, I'm not sure how it could be implemented, but from a user perspective, having a command like "Quarto Exporter: Render with Quarto" would be ideal. This command could first export to .qmd, then render directly into the format specified in the frontmatter, such as pptx, pdf, or docx, while discarding the intermediate .qmd file. The goal would be to get the final output format in a single step.
